### PR TITLE
Add documentation for Filter parameter format

### PR DESCRIPTION
**Pull request by lindgrenj6**
Add some filter examples so we can refer to here from catalog/approval/topology/sources etc.

### CODE REVIEW

**Comment by gmcculloug** [comment]
@abellotti Do you like or do you like?  😃

**Comment by abellotti** [comment]
for this I'd prefer to see a 2 column table showing the "Query Parameter" for the first and "Ruby Client Parameter" for the second. it would be easier to see.  Thanks!!

**Comment by lindgrenj6** [comment]
@abellotti updated!

File: README.md [hunk 1]
```diff
@@ -29,6 +29,26 @@ Or install it yourself as:
 
 TODO: Write usage instructions here
 
+#### ManageIQ::Api::Common::Filter
```
Comment (syncrou) [inline README.md]: We can probably remove the TODO on line 30 since you have instructions here for one part of the gem.

Comment (lindgrenj6) [reply README.md]: done!

**Pull request merged**